src/lib/menu/menu-item.ts

```diff
@@ -44,7 +44,7 @@ export class MdMenuItem implements Focusable {
   }
 
   /** Used to set the HTML `disabled` attribute. Necessary for links to be disabled properly. */
-  _getDisabledAttr(): boolean {
+  _getDisabledAttr(): true | null {
```

Comment:
I'm a fan of doing `disabled || null` in the host binding

This means that the attribute is correctly removed in the DOM but programmatic APIs return the expected `true`/`false`